### PR TITLE
MS14398: Make tutorial not grabbable

### PR DESCRIPTION
**Pull request by misslivirose**
PR to master for #13449

### CODE REVIEW

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/10849/HighFidelity-Beta-PR13450-2fe9913.apk

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/10849/HighFidelity-Beta-PR13450-2fe9913.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/10849/HighFidelity-Beta-Interface-PR13450-2fe9913.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/10849/HighFidelity-Beta-PR13450-2fe9913.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/10849/HighFidelity-Beta-Interface-PR13450-2fe9913.exe

**Comment by qliejahner** [comment]
Currently testing

**Comment by qliejahner** [comment]
PASS @conklin94122 

The detectors are now locked, and cannot be moved

**Comment by qlimnavarro** [comment]
Currently Testing

**Comment by qlimnavarro** [comment]
Pass @conklin94122 
Portal is Locked

File: cmake/externals/serverless-content/CMakeLists.txt [hunk 1]
```diff
@@ -4,8 +4,8 @@ set(EXTERNAL_NAME serverless-content)
 
 ExternalProject_Add(
   ${EXTERNAL_NAME}
-  URL http://cdn.highfidelity.com/content-sets/serverless-tutorial-RC68-v2.zip
-  URL_MD5 f7d290471baf7f5694c147217b8fc548
+  URL http://cdn.highfidelity.com/content-sets/serverless-tutorial-RC69.zip
+  URL_MD5 e2467b08de069da7e22ec8e032435592
```
Comment (huffman) [inline cmake/externals/serverless-content/CMakeLists.txt]: This looks good. Was the content manually updated off of RC68-v2?

Comment (misslivirose) [reply cmake/externals/serverless-content/CMakeLists.txt]: Yes. I just edited the models file from the previous version

Comment (huffman) [reply cmake/externals/serverless-content/CMakeLists.txt]: 👍 approved

**Pull request merged**